chapter2/tests/t20/terraform/dev/ec2.tf

```diff
@@ -0,0 +1,37 @@
+module "ec2_instances" {
```

Comment:
not semantic name. In your infra you may have multiple ec2 instances for different components of the project. APIs, DBs etc. They should be properly named semantically, which means `prefix` again.

`wordpress_ec2_instance` would be a good candidate

don't delete/recreate - move terraform state in the new name instead.